_data/sites.yml

```diff
@@ -978,6 +978,11 @@
   size: 37.6
   last_checked: 2022-08-01
 
+- domain: fz1.space
+  url: https://fz1.space/
+  size: 4.5
```

Comment:
Just to be fair with everyone else: could you change this to 4.59?

Reply:
okay